helm-charts/manual/postgresql/install.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
You can omit the charts in manual/ when using subcharts. Installing GISIB can then be a one-liner with Helm.